<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="2" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-17T17:39:14 IST" name="Suite" finished-at="2022-12-17T17:41:04 IST" duration-ms="110293">
    <groups>
    </groups>
    <test started-at="2022-12-17T17:39:14 IST" name="Test" finished-at="2022-12-17T17:41:04 IST" duration-ms="110293">
      <class name="com.edusoln.runner.TestNGRunner">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:14 IST" name="beforeTest" finished-at="2022-12-17T17:39:15 IST" duration-ms="327" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:15 IST" name="browserSetUp" finished-at="2022-12-17T17:39:33 IST" duration-ms="17516" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:33 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:39:33 IST" duration-ms="230" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{User id=Admin, Execute=No, TestCaseObjective=Verify Login works fine, AddEmployee=No, FirstName=No, LastName=No, MiddleName=No, EmpiID=No, Test case no=1, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Test case is not run ]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Test case is not run 
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:33 IST" name="tearDown" finished-at="2022-12-17T17:39:34 IST" duration-ms="783" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:34 IST" name="browserSetUp" finished-at="2022-12-17T17:39:52 IST" duration-ms="18134" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:39:52 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:40:17 IST" duration-ms="24995" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{User id=Admin, Execute=Yes, TestCaseObjective=Verify Add employee works fine, AddEmployee=Yes, FirstName=Sachin, LastName=Tendulkar, MiddleName=Ramesh, EmpiID=100, Test case no=2, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false],
expected [Persona details] but found [Personal Details]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [Persona details] but found [Personal Details]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:40:17 IST" name="tearDown" finished-at="2022-12-17T17:40:20 IST" duration-ms="2617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="browserSetUp()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:40:20 IST" name="browserSetUp" finished-at="2022-12-17T17:40:37 IST" duration-ms="16685" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserSetUp -->
        <test-method signature="testcase(java.util.Map)[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:40:37 IST" name="testcase" data-provider="dp" finished-at="2022-12-17T17:41:03 IST" duration-ms="25872" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{User id=Admin, Execute=Yes, TestCaseObjective=Verify Add employee works fine, AddEmployee=Yes, FirstName=Sourav, LastName=Ganguly, MiddleName=test, EmpiID=77, Test case no=3, Password=admin123}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [true] but found [false],
expected [Persona details] but found [Personal Details]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [Persona details] but found [Personal Details]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.edusoln.runner.TestNGRunner.testcase(TestNGRunner.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testcase -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:41:03 IST" name="tearDown" finished-at="2022-12-17T17:41:04 IST" duration-ms="816" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.edusoln.runner.TestNGRunner@7876d598]" started-at="2022-12-17T17:41:04 IST" name="afterTest" finished-at="2022-12-17T17:41:04 IST" duration-ms="618" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.edusoln.runner.TestNGRunner -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
